README.md

```diff
@@ -28,10 +28,39 @@ Execute this to update the profile:
 conan profile update settings.compiler.libcxx=libstdc++11 default
 ```
 
-### Buid & test & run:
+3. Install `conan`:
```

Comment:
It can be misleading, we are not installing Conan here, but the requirements specified in a recipe.
```suggestion
3. Install dependencies:
```